src/core/include/openvino/op/prior_box.hpp

```diff
@@ -66,5 +66,66 @@ class OPENVINO_API PriorBox : public Op {
     Attributes m_attrs;
 };
 }  // namespace v0
+
+namespace v8 {
+/// \brief Layer which generates prior boxes of specified sizes
+/// normalized to input image size
+class OPENVINO_API PriorBox : public Op {
+public:
+    OPENVINO_OP("PriorBox", "opset8");
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
We don't need this `BWDCMP_RTTI_DECLARATION` macro

Reply:
will  remove this macro in  next PR